.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: php
+
+sudo: false
+
+matrix:
+    include:
+        - php: 5.4
+        - php: 5.5
+        - php: 5.6
+        - php: 7.0
+        - php: hhvm
+    allow_failures:
+        - php: hhvm
+    fast_finish: true
+
+before_install:
+    - composer self-update
+
+install:
+    - composer --prefer-source install
+
+script:
+    - ezpublish/yaml_to_content_type/vendor/phpunit/phpunit/phpunit
```

Comment:
@valisj How should composer be defined after the restructuring in [comment #156817339](https://github.com/transfer-framework/examples/pull/1#issuecomment-156817339)?